### PR TITLE
Move qpstest.proto to a common area (test/proto) instead of test/cpp/qps

### LINKED ISSUE #3553: Move grpc/test/cpp/qps/qpstest.proto to a shared location
qpstest.proto will be used for other languages than just cpp, it should move to a common location (grpc/test/proto) before people start depending on it.

### PR DESCRIPTION
**Pull request by vjpai**
Fixes #3553

### CODE REVIEW

File: test/cpp/qps/perf_db.proto [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 syntax = "proto3";
```
Comment (jtattermusch) [inline test/cpp/qps/perf_db.proto]: PerfDB won't be needed by other languages than just cpp, right?

**Comment by jtattermusch** [comment]
LGTM (and one question).

File: test/cpp/qps/perf_db.proto (hunk 1 above)
Comment (vjpai) [inline test/cpp/qps/perf_db.proto]: It's just a tool, not a test, so I think it's ok in C++ alone.

Comment (vjpai) [inline test/cpp/qps/perf_db.proto]: Just confirming that perf_db is only used by the driver as an alternate reporting mechanism, not by the qps_worker (or any other solo client or server), so this should be fine. Will reconsider if needed.

**Pull request merged**